### DISCUSSION TITLE
Meaning of strings at the start of font names?

### DISCUSSION THREAD

**Discussion by np00** [jsvine/pdfplumber#603] [Q&A] [answered]
Hi all, 

great work! I really like this lib. 

My question is: 

**What is the meaning of the characters in front of the + symbol?**

```
FVCNHD+TimesNewRomanPS-BoldMT
LEWEVC+CMBX12
SEUPIM+CMBX12
MLKXRR+CMBX12
QZSPEU+CMBX12
YRXJEJ+CMBX12
VAKUSO+TimesNewRomanPS-BoldMT
ELKRUT+TimesNewRomanPS-BoldMT
AMRNLQ+TimesNewRomanPS-BoldMT
CKBTJY+CMBX12
BCDEEE+Times,Bold
TYLEBW+TimesNewRomanPS-BoldMT
DGXDXL+CMBX12
TQCRJK+NimbusRomNo9L-Medi
UCLFIE+TimesNewRomanPS-BoldMT
XRMABG+CMBX12
BLMIAK+CMBX12
CIOIPD+CMBX12
```

**Comment by jsvine** [comment, marked as answer]
Hi @np00, those identifiers come from the PDF itself. This StackOverflow answer has a good overview: https://tex.stackexchange.com/a/156438

Here's the relevant excerpt, from the [PDF specification](https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf) quoted there: 

> For a font subset, the PostScript name of the font—the value of the font’s BaseFont entry and the font descriptor’s FontName entry— shall begin with a tag followed by a plus sign (+). The tag shall consist of exactly six uppercase letters; the choice of letters is arbitrary, but different subsets in the same PDF file shall have different tags.
